docs/_includes/api/overview.html

```diff
@@ -28,7 +33,26 @@
 {% endmarkdown %}
 {% include alert/end.html%}
 
-For more info, check out the [guide to asynchronous code](/guides/async-code.html).
+To use a custom promise implementation with PouchDB, you must redefine a global `Promise` object before loading PouchDB:
+
+{% highlight html %}
+<script>window.Promise = MyCustomPromiseLibrary;</script>
+<script src="path/to/pouchdb.js"></script>
+{% endhighlight %}
+
+### Async functions
+
+If you are using a transpiler like [Babel](http://babeljs.io/), you can enable [async functions](http://babeljs.io/docs/plugins/syntax-async-functions/), which are an experimental API tentatively slated for release in ES7 (ES2016). This allows you to use the `async`/`await` keywords when consuming promise-based APIs like PouchDB's.
```

Comment:
Actually, that plugin only parses async functions, it doesn't convert them. You need https://babeljs.io/docs/plugins/transform-async-to-generator/ for that. (Or my plugin [kneden](https://github.com/marten-de-vries/kneden), but that's not production ready yet...)